sandbox/Devis/Main.tf

```
@@ -0,0 +1,44 @@
+locals {
+  env = "sit"
+  default_tags = {
+    env                = local.env
+    systemNumber       = "123452"
+    billingDescription = "Sandbox"
+    billingContact     = "sukhdeep.singh@posten.no"
+    billingUnit        = "123456"
+    creationSource     = "terraform"
+  }
+  region  = "norwayeast"
+  rg-main = "${local.env}-rg-sandbox-devis"
```

Always use _ (snake case) in terraform variable names and terraform resource names.. as it make it easy to format and easy to copy
```suggestion
  rg_main = "${local.env}-rg-sandbox-devis"
```